.agents/skills/openspec-workflows/references/create-change-from-plan.md

```
@@ -0,0 +1,312 @@
+- **GitHub Issue**: #<number>
+- **Issue URL**: <url>
+- **Last Synced Status**: proposed
+```
+
+5. Cleanup temp file.
+
+## Step 8: Completion
+
+Display summary:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)